resources/views/join.blade.php

```diff
@@ -31,12 +31,25 @@
 @endif
     <form action="/join" class="" method="POST" enctype="multipart/form-data" id="form">
       {{ csrf_field() }}
+      <!-- First Name Field -->
       <div class="field">
         <label class="label">Name</label>
         <div class="control">
-          <input id="name" class="input" type="text" placeholder="First Last" name="name" onchange="hideErrors()">
+          <input id="name" class="input" type="text" placeholder="First Name" name="name" onchange="hideErrors()">
```

Comment:
```suggestion
          <input id="name" class="input" type="text" placeholder="First Name" name="name" onchange="hideErrors()" required>
```
I forgot this as well on my version.